### ISSUE TITLE
NPM lock file not in sync(?) with package file

### ISSUE THREAD

**Issue by vsund** [blockstack/blockstack-browser#653]
`npm install` fails for me since `package-lock.json` [depends on a locally installed version](https://github.com/blockstack/blockstack-browser/blob/master/package-lock.json#L2642) of `blockstack.js`.

**Comment by guylepage3** [comment]
hm.. weird. this is happening on blockstack.org as well.

**Comment by larrysalibra** [comment]
@guylepage3 @shea256 the problem here is that i edited the `package.json` file manually instead of using `npm` to edit it - according to the `package-lock.json` documentation,

> package-lock.json is automatically generated for any operations where npm modifies either the node_modules tree, or package.json 

What likely happened was that I edited package.json to switch back to the deployed version of blockstack.js and commit the change before running `npm install` on my local development tree to update the `node_modules` and `package-lock.json` file. 

I've pushed a fix to `develop` which will make it into the next release (hopefully tomorrow).

The temporary work around for `master` is to delete package-lock.json from your local tree before installing.
